core-web/libs/edit-content/src/lib/utils/functions.util.spec.ts

```diff
@@ -660,4 +665,49 @@ describe('Utils Functions', () => {
             });
         });
     });
+
+    describe('generatePreviewUrl', () => {
```

Comment:
In order to avoid using an unknown concept like DotCMSContentlet, I created a utility to generate contentlet mocks, which I named createFakeContentlet, a utility imported from the @dotcms/utils-testing library. I believe this presents an excellent opportunity for practical application.

Example:

const contentLet = createFakeContentlet({
    URL_MAP_FOR_CONTENT: '/blog/post/5-snow-sports-to-try-this-winter',
    host: '48190c8c-42c4-46af-8d1a-0cd5db894797',
    languageId: 1
});

Reply:
Used.